lib/galaxy/datatypes/sequence.py

```diff
@@ -357,6 +357,12 @@ def sniff( self, filename ):
                         line = fh.readline().strip()
                         if line == '' or line.startswith( '>' ):
                             break
+                        
+                        # The third line may not contain chars like '()[].' otherwise it's most likely a DotBracket file                        
+                        line = fh.readline()
```

Comment:
The third line can also be a new header line which in turn can contain such characters.
I think this would be safer:
```
if not line.startswith( '>' ) and re.search("[\(\)\[\]\.]",line):
```